compat/src/suspense.js

```diff
@@ -73,13 +79,18 @@ Suspense.prototype._childDidSuspend = function(promise, suspendingComponent) {
 	};
 
 	const onSuspensionComplete = () => {
-		if (!--c._suspensions) {
+		if (!--c._pendingSuspensionCount) {
 			c._vnode._children[0] = c.state._suspended;
 			c.setState({ _suspended: (c._detachOnNextRender = null) });
+
+			let suspended;
+			while ((suspended = c._suspenders.pop())) {
+				suspended.forceUpdate();
```

Comment:
Note: I think this is a no-op if no sCU component is between Suspense and Lazy cuz `process()` will sort the render such that Suspense and all its children are diffed first, then when the Lazy forceUpdate are handled, the component will no longer be marked "_dirty" so they will be skipped